packages/components/src/utils/src/useFontFaceReady.ts

```diff
@@ -0,0 +1,17 @@
+import { useIsomorphicLayoutEffect } from "@hopper-ui/styled-system";
```

Comment:
whats the use of this?

Reply:
so that we can use useLayoutEffect, but it'll default to useEffect for server side rendering.